<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HttpPostWidget</class>
 <widget class="QWidget" name="HttpPostWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="tabBtn_background" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#tabBtn_background{
background-color: rgb(255, 255, 255);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_cli">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>client</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_serv">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>server</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_cli">
      <property name="styleSheet">
       <string notr="true">QWidget#page_cli{
background-color: rgb(255, 255, 255);
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEdit_11">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
background-color: rgb(240, 240, 240);
border-radius:3px;
}
</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_12">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
background-color: rgb(240, 240, 240);
border-radius:3px;
}
</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_15">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
border-radius:5px
;background-color: rgb(205, 232, 255);
}

QPushButton:hover {
background-color: rgb(255, 165, 0);
}</string>
           </property>
           <property name="text">
            <string>连接</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_16">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
border-radius:5px
;background-color: rgb(205, 232, 255);
}

QPushButton:hover {
background-color: rgb(255, 165, 0);
}</string>
           </property>
           <property name="text">
            <string>断开</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_17">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 85, 0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_14">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
background-color: rgb(240, 240, 240);
border-radius:5px;
}
</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_10">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
border-radius:5px
;background-color: rgb(205, 232, 255);
}

QPushButton:hover {
background-color: rgb(255, 165, 0);
}</string>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="plainTextEdit_7">
         <property name="styleSheet">
          <string notr="true">QPlainTextEdit{
background-color: rgb(240, 240, 240);
border-radius:5px;
}
</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="lineEdit_13">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
background-color: rgb(240, 240, 240);
border-radius:3px;
}
</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_11">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
border-radius:5px
;background-color: rgb(205, 232, 255);
}

QPushButton:hover {
background-color: rgb(255, 165, 0);
}</string>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="plainTextEdit_8">
         <property name="styleSheet">
          <string notr="true">QPlainTextEdit{
background-color: rgb(240, 240, 240);
border-radius:5px;
}
</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_serv">
      <property name="styleSheet">
       <string notr="true">QWidget#page_serv{
background-color: rgb(255, 255, 255);
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" rowspan="2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLineEdit" name="lineEdit_servPort">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
background-color: rgb(240, 240, 240);
border-radius:3px;
}
</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btn_servStart">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
border-radius:5px
;background-color: rgb(205, 232, 255);
}

QPushButton:hover {
background-color: rgb(255, 165, 0);
}</string>
           </property>
           <property name="text">
            <string>启动</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_servStop">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
border-radius:5px
;background-color: rgb(205, 232, 255);
}

QPushButton:hover {
background-color: rgb(255, 165, 0);
}</string>
           </property>
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QPlainTextEdit" name="plainTextEdit_servMsg">
           <property name="styleSheet">
            <string notr="true">QPlainTextEdit{
background-color: rgb(240, 240, 240);
border-radius:5px;
}
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_servUrl_1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
background-color: rgb(240, 240, 240);
border-radius:3px;
}
</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPlainTextEdit" name="plainTextEdit_rsqMsg_1">
         <property name="styleSheet">
          <string notr="true">QPlainTextEdit{
background-color: rgb(240, 240, 240);
border-radius:5px;
}
</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="lineEdit_servUrl_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
background-color: rgb(240, 240, 240);
border-radius:3px;
}
</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_servUrl_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>227215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
background-color: rgb(240, 240, 240);
border-radius:3px;
}
</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit_rsqMsg_2">
         <property name="styleSheet">
          <string notr="true">QPlainTextEdit{
background-color: rgb(240, 240, 240);
border-radius:5px;
}
</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPlainTextEdit" name="plainTextEdit_rsqMsg_3">
         <property name="styleSheet">
          <string notr="true">QPlainTextEdit{
background-color: rgb(240, 240, 240);
border-radius:5px;
}
</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
